docs/content/geoh5_format/analyst/objects.rst

```diff
@@ -0,0 +1,476 @@
+ANALYST Objects
+===============
+
+*To be further documented*
+
+Points
+------
+
+**UUID : {202C5DB1-A56D-4004-9CAD-BAAFD8899406}**
+
+*To be further documented*
+
+Object defined by vertices only - no cell data.
+
+Curve
+-----
+
+**UUID : {6A057FDC-B355-11E3-95BE-FD84A7FFCB88}**
+
+*To be further documented*
+
+Each cell contains two vertex indices, representing a segment.
+
+Surface
+-------
+
+**UUID : {F26FEBA3-ADED-494B-B9E9-B2BBCBE298E1}**
+
+*To be further documented*
+
+Each cell contains three vertex indices, representing a triangle.
+
+Block model
+-----------
+
+**UUID : {B020A277-90E2-4CD7-84D6-612EE3F25051}**
+
+*To be further documented*
+
+Each cell represents a point of a 3D rectilinear grid. For a 3D cell
+index (i,j,k) along axes U,V and Z of length nU, nV and nZ respectively,
+
+::
+
+   cell index = k + i*nZ + j*nU*nZ
+
+- Without rotation angles, U points eastwards, V points northwards, and
+   Z points upwards.
```

Comment:
In html the first line is bold and the second indented line is not.  Is this intended?

Reply:
... it's like this all through the document

Reply:
good catch, will change all to plain text